### ISSUE TITLE
accessibility problem with screen reader

### ISSUE THREAD

**Issue by ghost** [mrdokenny/Cookie-AutoDelete#610]
Hi,
this is a screen reader user.
After installing your addon on firefox/chrome, on the page cookie setting.
screen reader cannot tell whether the option is checked or not.

for example it said "clickable notify me after cookie deletion"
no matter if I activate it with space or not, it also speak the same thing, and I cannot determine whether it is activated or not.

**Comment by kennethtran93** [comment]
Thanks for the issue.  We'll look into it at a future time.  I cannot give any estimations at this time.

**Comment by kennethtran93** [comment]
work in progress.  I cannot guarantee a fully working one but will try.  

Also the space will most likely NOT do anything as it appears the backend code only looks for 'clicks' and not 'keypress'.

**Comment by kennethtran93** [comment]
#713 has small accessibility enhancements in regards to the checkboxes.  I've attempted to add role="checkbox" and use aria-checked value for them.

Don't think I made it accessible by spacebar yet though.
